core/src/utilities/helpers/navigation-helpers.js

```diff
@@ -114,8 +115,8 @@ class NavigationHelpersClass {
       let key;
       let metaInfo;
       const category = node[property];
-      if (category && typeof category === 'object') {
-        key = category.label;
+      if (category && GenericHelpers.isObject(category)) {
```

Comment:
```suggestion
      if (GenericHelpers.isObject(category)) {
```
`isObject` function already checked if item exists